clients/vscode-hlasmplugin/package.json

```diff
@@ -245,6 +246,11 @@
         "command": "rearrangeSequenceNumbers",
         "category": "HLASM",
         "title": "RearrangeSequenceNumbers"
+      },
+      {
+        "command": "extension.hlasm-plugin.downloadDependencies",
```

Comment:
What's the reason behind naming the command with the prefix `extension.hlasm-plugin.`? Should other commands be renamed as well to maintain consistency?

Reply:
I think the primary concern was uniqueness... I'll change the others.